.github/workflows/run-osp-extended-tests.yml

```diff
@@ -0,0 +1,147 @@
+          github.event.comment.author_association == 'MEMBER' ||
+          github.event.comment.author_association == 'COLLABORATOR')
+        )
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get PR branch
+        uses: xt0rted/pull-request-comment-branch@v1
+        id: comment-branch
+        if: github.event_name == 'issue_comment'
+      - name: Set latest commit status as pending
+        if: github.event_name == 'issue_comment'
+        uses: myrotvorets/set-commit-status-action@master
+        with:
+          sha: ${{ steps.comment-branch.outputs.head_sha }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          status: pending
+
+      - name: Checkout forklift
+        uses: actions/checkout@v3
+
+      - name: prepare the secrets
+#        uses: kubev2v/forkliftci/ci/prepare-ansible-secrets@v5.0
+        uses: eslutsky/forkliftci/ci/prepare-ansible-secrets@validate-ssh-key
+        with:
+          OKD_ENGINE_SECRETS: ${{ secrets.OKD_ENGINE_SECRETS }}
+          OKD_SSH_KEY: ${{ secrets.OKD_SSH_KEY }}
+
+      - name: provision self hosted runner
+        #uses: kubev2v/forkliftci/ci/create-self-runner@v5.0
+        uses: eslutsky/forkliftci/ci/create-self-runner@validate-ssh-key
+
+      - name: Set latest commit status as ${{ job.status }}
+        uses: myrotvorets/set-commit-status-action@master
+        if: ${{ failure() || cancelled() }}
+        with:
+          sha: ${{ steps.comment-branch.outputs.head_sha }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+          status: ${{ job.status }}
+
+  ci_setup:
+    runs-on: self-hosted
+    needs: create_ovirt_runner
+    env:
+      USE_BAZEL_VERSION: 5.4.0
+      source_provider: openstack
+    timeout-minutes: 45
+    steps:
+      - name: Checkout forklift
+        uses: actions/checkout@v3
+        with:
+          path: forklift
+
+      - name: Checkout forkliftci
+        uses: actions/checkout@v3
+        with:
+          repository: kubev2v/forkliftci
+          ref: v5.0
+
+      - name: add docker-hub token
+        run: echo "${{ secrets.DOCKER_HUB_TOKEN }}" | docker login --username "${{  secrets.DOCKER_HUB_USER }}" --password-stdin
+
+      - name: build and setup everything
+        uses: kubev2v/forkliftci/ci/build-and-setup@v5.0
+        with:
+          provider_name: openstack
+          gh_access_token: ${{ secrets.GITHUB_TOKEN }}
+          enable_openstack_ssl: true
+
+      - run: kubectl version
+
+      - run: kubectl get pods -n konveyor-forklift
+
+      - name: run e2e-sanity-openstack
+        uses: kubev2v/forkliftci/ci/run-suite@v5.0
+        with:
+          suite_name: e2e-sanity-openstack-extended
+
+      - name: save k8s logs and upload-artifact
+        if: ${{ always() }}
+        uses: kubev2v/forkliftci/ci/save-artifacts@v5.0
+        with:
+          source_provider: openstack
+
+      - name: Get PR branch
+        if: always() && github.event_name == 'issue_comment'
```

Comment:
why doing this rather than just `if: github.event_name == 'issue_comment'`? (here and below)

Reply:
we want this action to be triggered regardless of the previous step result (succees or failure),
according to the [docs](https://docs.github.com/en/actions/learn-github-actions/expressions#status-check-functions) 
```
You can use the following status check functions as expressions in if conditionals. A default status check of success() is applied unless you include one of these functions.
```
Where “one of these functions” refers to:
- sucess()
- always()
- cancelled()
- failure()

Reply:
got it